src/markdown_generators.js

```diff
@@ -152,7 +152,8 @@ const generateMarkdownFromJson = courseData => {
   const rootSections = courseData["course_pages"].filter(
     page =>
       page["parent_uid"] === courseData["uid"] &&
-      page["type"] !== "CourseHomeSection"
+      page["type"] !== "CourseHomeSection" &&
+      page["type"] !== "DownloadSection"
```

Comment:
part of https://github.com/mitodl/ocw-to-hugo/pull/54, right?

Reply:
It won't, this was just a failure on my part to properly base this branch off of master when I created it.  It might be less problematic git-wise to merge #54  first though and then rebase this.